packages/jobs/lib/execution/sync.ts

```diff
@@ -216,6 +216,11 @@ export async function handleSyncSuccess({ nangoProps }: { nangoProps: NangoProps
             provider_config_key: nangoProps.providerConfigKey
         };
 
+        const providerConfig = await configService.getProviderConfig(connection.provider_config_key, connection.environment_id);
+        if (providerConfig === null) {
+            throw new Error(`Provider config not found for connection: ${connection.connection_id} in handleSyncSuccess`);
```

Comment:
Not sure it's required since we have the stack trace 
```suggestion
            throw new Error(`Provider config not found for connection: ${connection.connection_id}`);
```